test_scripts/WebEngine/WebEngineProjection/009_Registration_reject_NON_WEBENGINE_APP.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that non-webengine App will be rejected with HMI type WEB_VIEW 
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+-- 3. PT contains record for App1
+
+-- Sequence:
+-- 1. Set application properties with endpoint parameter
+-- 1. Application try to register with WEB_VIEW appHMIType
+--  a. SDL reject registration of application (resultCode DISALLOWED, success:"false")
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
```

Comment:
@YarikMamykin I would propose to write similar test scripts with the same style to avoid confusion for reviewers
```suggestion
local appHMIType = { "WEB_VIEW" }
```

Reply:
@aderiabin fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/66e780b7fb2d744f4ac6fd6ee635b18b1bf5b68f

Reply:
@aderiabin  fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/78f85afd888bf843ab07acfddfe3d2f39f8a23fc